frontend/README.md

```
@@ -1,112 +1,220 @@
+
+```typescript
+import { safeValidate, withdrawalSchema } from "@/lib/validation";
+
+const result = safeValidate(withdrawalSchema, userInput);
+if (!result.success) {
+  setError(result.error);
+  return;
+}
+```
+
+#### 3. Client-side CSRF (REQUIRED)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)